src/main/kotlin/racingcar/view/InputViewImpl.kt

```diff
@@ -0,0 +1,5 @@
+package racingcar.view
+
+class InputViewImpl : InputView {
+    override fun readInput(): String = readLine()!!
```

Comment:
```suggestion
    override fun readInput(): String = readln()
```
[readln](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/readln.html) 함수를 사용하면 not nullable한 리턴을 받을 수 있습니다!